base/commands/migration/utils.go

```diff
@@ -119,6 +119,14 @@ func readPathAsString(path string) (string, error) {
 	return string(b), nil
 }
 
-func makeMigrationID() string {
+func MakeMigrationID() string {
 	return types.NewUUID().String()
 }
+
+func MakeStatusMapName(migrationID string) string {
+	return "__datamigration_" + migrationID
```

Comment:
you could make this string a constant.

Reply:
https://github.com/hazelcast/hazelcast-commandline-client/pull/367/commits/d0dfafa7c167f535e3ba6703862c0d9ea14dc7b6